quaint/src/visitor/postgres.rs

```diff
@@ -17,6 +17,24 @@ pub struct Postgres<'a> {
     parameters: Vec<Value<'a>>,
 }
 
+impl<'a> Postgres<'a> {
+    fn visit_json_build_obj_expr(&mut self, expr: Expression<'a>) -> crate::Result<()> {
+        dbg!(&expr);
+        match expr.kind() {
+            ExpressionKind::Column(col) => match (col.type_family.as_ref(), col.native_type.as_deref()) {
+                (Some(TypeFamily::Decimal(_)), Some("MONEY")) => {
```

Comment:
IIRC the native type always comes in this capitalization. But can you please double check, or otherwise force upcasing it in the schema parsing pass?

Reply:
It's always uppercased specifically in the `NativeColumnType` type because you made it this way:

https://github.com/prisma/prisma-engines/blob/6b41f7a3b9f16da11f37cf41d70e8149f95eb6cf/quaint/src/ast/values.rs#L48-L52

It sucks that it makes us do an allocation but it's convenient.